src/js/index.js

```diff
@@ -5,11 +5,25 @@ import * as shareUtils from './share/utils';
 import Blockly from "blockly";
 import { renderAllPrograms } from './render/render';
 
-(() => {
+(async () => {
   if (process.env.NODE_ENV === 'development') {
     window.Blockly = Blockly;
   }
 
+  let language = 'en';
+  if (process.env.DISPLAY_LANGUAGE !== undefined && process.env.DISPLAY_LANGUAGE.length > 0) {
```

Comment:
found bug after code review (setting language via docker was not working) and fixed it:
moved environment variable check before the language is set by `await Blockly.CatblocksMsgs.setLocale(language);`